tests/test_loan_transitions.py

```diff
@@ -335,6 +337,80 @@ def test_checkout_on_unavailable_item(
         )
 
 
+def _loan_steps_created_to_on_loan(loan_created, params):
+    # go step by step through all the transitions
+
+    # loan created
+    loan = current_circulation.circulation.trigger(
+        loan_created, **dict(params, trigger="request")
+    )
+
+    # loan pending
+    loan = current_circulation.circulation.trigger(
+        loan_created, **dict(params, trigger="next")
+    )
+
+    # item at desk
+    loan = current_circulation.circulation.trigger(
+        loan_created, **dict(params)
+    )
+
+
+def test_checkout_item(loan_created, db, params, mock_is_item_available,
+                       users, app):
+    login_user(users['user'])
+    # set specific loan owner
+    params["patron_pid"] = "3"
+    _loan_steps_created_to_on_loan(loan_created, params)
+
+
+def test_checkout_item_by_manager(loan_created, db, params,
+                                  mock_is_item_available,
+                                  users, app):
+    user = users['manager']
```

Comment:
Manager -> librarian or admin?

Reply:
manager=librarian, I think that was an initial idea in the conftest so I continued with that naming